core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -102,6 +104,15 @@ public static void setupContent() throws FileNotFoundException {
             .withCommand("/bin/sh", "-c", "while true; do cat /content.txt | nc -l -p 80; done");
 
     /**
+     * Map a file on the classpath to a file in the container, and then expose the content for testing.
+     */
+    @ClassRule
+    public static GenericContainer alpineClasspathResourceSelinx = new GenericContainer("alpine:3.2")
+            .withExposedPorts(80)
+            .withClasspathResourceMapping("mappable-resource/test-resource.txt", "/content.txt", READ_WRITE, SHARED)
+            .withCommand("/bin/sh", "-c", "while true; do cat /content.txt | nc -l -p 80; done");
```

Comment:
True story.  Not sure how I'd actually go about doing that other than inspecting the assigned seLinux type in the running container or on the host machine.  Any suggestions as to how I might inspect that?  I'm assuming we don't want our test to call 'inspect'...